anta/runner.py

```diff
@@ -30,16 +33,25 @@ async def main(
         inventory (AntaInventory): Device inventory object.
         tests (List[...]): Test catalog. Output of anta.loader.parse_catalog().
 
+    Example:
+        anta.tests.aaa:
+          - VerifyAAAMethods:
+              method: login
+              tpl_options:
+                - stage: authentication
+                - stage: authorization
```

Comment:
I think we should put the BGP example here since it's a real example. The VerifyAAAMethods test doesn't exist, I created multiple AAA tests instead.